c8/fabric.py

```diff
@@ -44,7 +44,7 @@
     EventDeleteError,
     EventGetError,
     StreamAppGetSampleError,
-    GetAPIKeys
+    GetAPIKeys, CollectionPropertiesError
```

Comment:
you can put the collectionpropertieserror in the a new line

Reply:
same, thanks!